tests/ignite/contrib/handlers/test_lr_finder.py

```diff
@@ -414,7 +414,7 @@ def test_no_matplotlib(no_site_packages, lr_finder):
 
 def test_plot_single_param_group(lr_finder, mnist_to_save, dummy_engine_mnist, mnist_dataloader):
 
-    with lr_finder.attach(dummy_engine_mnist, mnist_to_save, end_lr=20, smooth_f=0.04) as trainer_with_finder:
+    with lr_finder.attach(dummy_engine_mnist, mnist_to_save, end_lr=100, smooth_f=0.04) as trainer_with_finder:
```

Comment:
Thanks for the updates! as this doesn't work, please try to manipulate `diverge_th`, this may solve this issue

Reply:
For more info about the `args`, please check the docs

Reply:
I think I will skip on the `lr_finder` issues since I had a quite bit of trouble tackling them. I feel I am not capable enough to handle them myself and someone better than me should pick this one up, sorry for the inconvenience.

Reply:
No problem, If you need more time/send a separate PR for `lr_finder`, I can help you with that, If you don't want to work on `lr_finder`, no problem! thanks for the updates!

Reply:
Sure thing! Thanks for all the help and guidance till now. I will keep the PR as it is and wait for further review, I have rolled back the possible changes back to their default state as of now. If anything clicks for me again I will make sure to drop another follow up PR and ask for help if need be. Thank you so much!